po/es.po

```diff
@@ -11271,23 +11273,23 @@ msgstr "desviación de tonos medios"
 
 #: ../src/iop/velvia.c:392
 msgid "the strength of saturation boost"
-msgstr "fuerza del incremento de saturación"
+msgstr "Intensidad del realce de la saturación"
```

Comment:
Totally subjective opinion: I don't really like the "realce" word, it's kinda over-elaborate.

Reply:
But that's what describes what the module does.